### PR TITLE
entity debugger and debug arrow

### PR DESCRIPTION
**Pull request by nesrak1**
That was scary but looks like we have debug stuff now so yay

### CODE REVIEW

File: NitroxClient/Debuggers/EntityDebugger.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+        private void RenderEntityDebugger()
+        {
+        }
+
+        public override void OnGUIExtra()
+        {
+            foreach (KeyValuePair<NitroxId, GameObject> gameObjectPairs in NitroxEntity.GetGameObjects())
+            {
+                NitroxId id = gameObjectPairs.Key;
```
Comment (Jannify) [inline NitroxClient/Debuggers/EntityDebugger.cs]: Do you need to initialize a new variable for a one time use?

File: NitroxClient/Debuggers/EntityDebugger.cs [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+                GameObject gameObject = gameObjectPairs.Value;
```
Comment (Jannify) [inline NitroxClient/Debuggers/EntityDebugger.cs]: If it's just a reference I think it's better to use `GameObject gameObject => gameObjectPairs.Value;`

File: NitroxClient/Debuggers/EntityDebugger.cs (hunk 1 above)
Comment (nesrak1) [reply NitroxClient/Debuggers/EntityDebugger.cs]: I guess not? Just seemed more consistent to have both vars up front.

Comment (Jannify) [reply NitroxClient/Debuggers/EntityDebugger.cs]: Then I think it would be the best to use `=>`

**Pull request merged**